cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/wizard-basic/management-credential-step/ManagementCredentials.tsx

```diff
@@ -76,15 +77,18 @@ function ManagementCredentials(props: Partial<StepProps>) {
             azureCloud: azureState[STORE_SECTION_FORM].AZURE_ENVIRONMENT,
         };
         try {
+            setStatus(CONNECTION_STATUS.CONNECTING);
+            setMessage('Connecting to Azure');
             await AzureService.setAzureEndpoint(params);
-            setConnection(true);
+            setStatus(CONNECTION_STATUS.CONNECTED);
+            setMessage('Connected to Azure');
             retrieveRegions();
-        } catch (e) {
-            console.log('Error');
+        } catch (err: any) {
+            setMessage(err.body.message);
```

Comment:
Please add some curated text to the beginning of the error message like: Unable to connect to Azure: ...error message here...

Reply:
fixed.